src/PHP8Fun.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * @param $a
+ * @param $b
+ * Call str_contains!
+ * @return bool
+ */
+function find_my_string( $a, $b ): bool {
+	if ( $x = str_contains( $a, $b ) ) {
```

Comment:
:warning: **Warning**: Variable assignment found within a condition. Did you mean to do a comparison? (*WordPress.CodeAnalysis.AssignmentInCondition.Found*).